src/Sylius/Bundle/UserBundle/Command/AbstractRoleCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Command;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Component\Core\Model\UserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Loïc Frémont <loic@mobizel.com>
+ */
+abstract class AbstractRoleCommand extends ContainerAwareCommand implements RoleCommandInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        if (!$input->getArgument('email')) {
+            $email = $this->getHelper('dialog')->askAndValidate(
+                $output,
+                'Please enter an email:',
+                function($email) {
+                    if (empty($email)) {
+                        throw new \Exception('Email can not be empty');
+                    }
+                    return $email;
```

Comment:
Still, there should be a blank line before return statement. [See](http://symfony.com/doc/current/contributing/code/standards.html#structure)